### ISSUE TITLE
Custom API URL Through Ng2Map Service

### ISSUE THREAD

**Issue by allenhwkim** [ng2-ui/ng2-map#3]
Google Changed its policy. 
We need to support API_KEY in url as well as any other options.

http://googlegeodevelopers.blogspot.ca/2016/06/building-for-scale-updates-to-google.html

We no longer support keyless access (any request that doesn't include an API key). Future product updates are only available for requests made with an API key. API keys allow us to contact developers when required and help us identify misbehaving implementations.

**Comment by kharatps02** [comment]
Yes, I'm also facing same issue.

**Comment by allenhwkim** [comment]
@kharatps02 thanks for trying this module. if you want to try to fix and make PR, I would be appreciate it. I will work on this issue later, but not sooner though.

**Comment by allenhwkim** [comment]
This feature is enabled and available with 0.3.1.

Example:

```
import { NG2_MAP_DIRECTIVES, Ng2MapComponent } from "ng2-map";  // <--- this

@Component({
  selector: 'my-app',
  directives: [ NG2_MAP_DIRECTIVES ],
  templateUrl: 'app/app.tpl.html'
})
export class AppComponent {
  public center ="Brampton, Canada";
  public positions=[];
  constructor() {
    Ng2MapComponent.apiUrl = "https://maps.google.com/maps/api/js?key=YOUR_KEY";
  }
```
